README.adoc

```diff
@@ -48,6 +76,39 @@ public interface ProductRepository extends PageableRepository<Product, Long> {
 }
 ----
 
+==== Type-safe Access to Entity Attributes
+
+Jakarta Data provides a static metamodel that allows entity attributes to be accessed by applications in a type-safe manner. This metamodel generates metadata classes that mirror your entity classes, enabling you to work with entity attributes using compile-time safety.
+
+With the Jakarta Data static metamodel, you can access entity attributes in a type-safe manner. Here's a quick example to demonstrate how it works:
+
+[source,java]
+----
+
+@Entity
+public class Product {
+  public long id;
+  public String name;
+  public float price;
+}
+----
+
+
+Usage example:
+
+[source,java]
+----
+
+List<Product> found = products.findByNameContains(searchPattern,
+                                                  Product_.price.desc(),
+                                                  Product_.name.asc(),
+                                                  Product_.id.asc());
+----
+
+
+In this example, `Product_` is the metadata class generated by Jakarta Data, allowing you to access entity attributes in a type-safe way. It helps prevent runtime errors related to attribute names and enables more reliable and efficient data access.
+
+
```

Comment:
There is a separate issue under which discussion is occurring about whether we want to change the StaticMetamodel.  We should avoid adding new sections like this while that is still being discussed in the other issue.  Also, this document already contains a section that is titled "Type-safe Access to Entity Attributes", and so we should consider putting the changes there when we do make them.

```suggestion
```